libbeat/docs/loggingconfig.asciidoc

```diff
@@ -63,11 +63,11 @@ errors, there will be no log file in the directory specified for logs.
 [[level]]
 ==== `logging.level`
 
-Minimum log level. One of `debug`, `info`, `warning`, or `error`. The default
+Minimum log level. One of `debug`, `info`, `warn`, or `error`. The default
```

Comment:
Remember to change this back to `warning`, as we went for making `warning` work again in https://github.com/elastic/beats/pull/6240

Reply:
fixed. Thanks for the reminder.